spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -589,3 +673,57 @@ def debug_command(self, command):
         if widget is None:
             return
         widget.shellwidget.pdb_execute_command(command)
+
+    def _load_data(self):
```

Comment:
```suggestion
    def load_data(self):
```

Renaming this to be public because it doesn't need to be private now.